src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -279,12 +278,12 @@ def tree_values_preprocessing(
             # Get the preprocessed tree predictions to replace the current (non-quantized)
             # values in the onnx_model.
             init_tensor = q_y.qvalues
-        else:
```

Comment:
this was wrong, ceil/floor should be applied only on bias_1 should (but it did not create any conflicts as other weights or bias are already quantized)